### ISSUE TITLE
Capture whole screen by keypress

### ISSUE THREAD

**Issue by Froloket64** [emersion/slurp#108]
It would be nice to have a feature to capture the whole screen by pressing a specific key (lets say `f` or `Return`) when the selection pops up.

**Comment by matejdro** [comment]
Maybe just clicking on the display would select the whole screen by default? Currently it cancels selection which seems a bit counter intuitive

**Comment by tmccombs** [comment]
> Maybe just clicking on the display would select the whole screen

That's what the "-o" option does

**Comment by matejdro** [comment]
But -o option then always selects whole display. It does not give me an option to either select a region or to select whole display.

**Comment by tmccombs** [comment]
> But -o option then always selects whole display. It does not give me an option to either select a region or to select whole display.

No, by default if you use the the `-o` option, then dragging a region will select the region, not the whole output. The behavior you are describing would require the `-r` option as well.

**Comment by matejdro** [comment]
Doh, thank you, it appears I have mixed the flags.
